services/minespace-web/src/styles/settings/theme.scss

```diff
@@ -45,25 +45,17 @@ $black: #000;
 
 // Color used by default to control hover and active backgrounds and for
 // alert info backgrounds.
-$primary-1: tint(
-  $primary-color,
-  90%
-); //color(~`colorPalette('${primary-color}', 1) `); // replace tint($primary-color, 90%)
-$primary-2: tint(
-  $primary-color,
-  80%
-); //color(~`colorPalette('${primary-color}', 2) `); // replace tint($primary-color, 80%)
+$primary-1: tint($primary-color,
```

Comment:
I think that all of the changes I made in `theme.scss` I ended up reverting in a later commit, so this is all eslint.